gauntlet/packages/gauntlet-serum-multisig/networks/.env.local

```diff
@@ -1 +1,10 @@
-NODE_URL=http://127.0.0.1:8899
\ No newline at end of file
+NODE_URL=http://127.0.0.1:8899
+
+PROGRAM_ID_MULTISIG=3X4UvrX9gTxSmNmXww53W5dM5fT1Nby246hKV8Mx9LuD
```

Comment:
Remove these local vars?

Reply:
Those are constant always on local environment

Reply:
PROGRAM_IDs yes, but MULTISIG_ADDRESS is created anew each time, no?

Reply:
That's right. Removing those